_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -944,13 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
 
-=== Configure the security baseline for the root account
+[[deploy_landingzone]]
+=== Deploy Landing Zone solution
+
+==== Apply the `account-baseline-root` to the root account
+
+===== Configure the `account-baseline-root` for the root account
 
 [.exceptional]
-IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-cis-service-catalog`.
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork Compliance subscriber# to access the Gruntwork Infrastructure as Code Library and the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/[CIS AWS Foundations Benchmark modules].
+
+First, let's set consider the repository structure that is recommended by this guide. It is available for your reference in the "/examples/for-production" section in the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. Consider the following directory structure for your `infrastructure-live` repository. It showcases the configuration files foryour local variables.
+
+[source]
+----
+.
+└ infrastructure-live
+    └ root
+        └ account.hcl
+        └ global
+            └ region.hcl
+        └ us-east-1
+            └ region.hcl
+    └ common.hcl
+    └ accounts.json
+----
+
+Each of the `region.hcl`, `account.hcl` and `common.hcl` should contain the relevant information, so in your modules you're able to reference the values like this:
+
+[source, hcl]
+----
+locals {
+  # Automatically load common variables shared across all accounts
+  common_vars = read_terragrunt_config(find_in_parent_folders("common.hcl"))
+
+  # Automatically load account-level variables
+  account_vars = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract the account_name for easy access
+  account_name = local.account_vars.locals.account_name
+
+  # Automatically load region-level variables
+  region_vars = read_terragrunt_config(find_in_parent_folders("region.hcl"))
+
+  # Extract the region for easy access
+  aws_region = local.region_vars.locals.aws_region
+}
+----
+
+You'll need to create these files in order to be able to follow the code examples following. For examples on what to put in each of these files, refer to the https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/for-production[`terraform-aws-cis-service-catalog` repository]. The `locals` configuration above can be used in each of the `terragrunt.hcl` files to help you avoid repetition. Note that the examples below won't show this in the interest of clarity.
 
-Next, we'll configure a security baseline for the root account that is responsible for creating all the child accounts.
+Next, we'll configure the `account-baseline-root` baseline for the root account that is responsible for creating all the child accounts.
```

Comment:
This phrase is a bit weird for me 🤔 

> the `account-baseline-root` baseline for the root account 

account, baseline and root appears two times each 😅  this is super NIT

Reply:
ahhhh, readability is indeed awful 😰 , to thanks for point it out to me!